.gitlab-ci.yml

```diff
@@ -336,23 +404,24 @@ deploy-polkasync-kusama:
     # former .rules-build
     - if: $CI_COMMIT_REF_NAME == "rococo-v1"
       when: never
-    - if: $CI_PIPELINE_SOURCE == "web"
     - if: $CI_PIPELINE_SOURCE == "schedule"
     - if: $CI_COMMIT_REF_NAME == "master"
-    - if: $CI_COMMIT_REF_NAME =~ /^v[0-9]+\.[0-9]+.*$/              # i.e. v1.0, v2.1rc1
   variables:
     POLKADOT_CI_COMMIT_NAME:       "${CI_COMMIT_REF_NAME}"
-    POLKADOT_CI_COMMIT_REF:        "${CI_COMMIT_REF}"
+    POLKADOT_CI_COMMIT_REF:        "${CI_COMMIT_SHORT_SHA}"
   allow_failure:                   true
   trigger:                         "parity/infrastructure/parity-testnet"
 
 trigger-simnet:
   stage:                           deploy
-  <<:                              *rules-test
+  # DAG from `build-adder-collator` job
```

Comment:
I'm not sure what this comment means. If DAG means "directed acyclic graph" here, then the sentence doesn't make a lot of sense. If it means something else, then I don't know the acronym. Could be explained better.

Reply:
It is a directed acyclic graph that starts on the mentioned job being dependent on whether if it included in the pipeline or not. Ok I will explain it a bit broader.